app/assets/stylesheets/components-vue/_carousel.scss

```diff
@@ -1,204 +1,90 @@
-//--------------------------------------------------
-// carousel animation
-//--------------------------------------------------
-@keyframes fadeOut {
-  0% { 
-    opacity: 1;
 
-  }
-  100% { 
-    opacity: 0; 
-    transform: translateY(-50%);
-  }
-}
-
-@keyframes fadeIn {
-  0% { 
-    opacity: 0;
+.carousel {
+  background-color: $black;
+  color: $white;
 
-  }
-  100% { 
-    opacity: 1;
-  }
-}
+  position: relative;
 
-@keyframes backgroundFadeOut {
-  0% { 
-    opacity: 1;
+  &__content {
+    padding-right: rem-calc(44);
 
+    @include breakpoint($small) {
+      padding-right: 50%;
+    }
   }
-  100% { 
-    opacity: 0;
-  }
-}
 
-//--------------------------------------------------
-// carousel
-//--------------------------------------------------
-.carousel {
-  
-//--------------------------------------------------
-// fixed
-//--------------------------------------------------
-  &--fixed {
-    color: $white;
-    background-color: rgba($black, 0.9);
-    padding-right: rem-calc(70);
-    width: 100%; min-height: 100vh;
-
-    position: relative;
-
-    @include breakpoint($small) { 
-      padding-right: 0;
-      height: 100vh;
-    }
+  &__nav {
+    position: absolute;
+    top: calc(100vh/2);
```

Comment:
Isn't this just 50vh or 50%? Asking because I'm not sure whether there's a difference in the output value from the calculation